pallets/schemas/src/lib.rs

```diff
@@ -0,0 +1,140 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use frame_support::{dispatch::DispatchResult, ensure, traits::Get, BoundedVec};
+use sp_runtime::{
+	traits::{IdentifyAccount, Verify},
+	DispatchError,
+};
+use sp_std::{convert::TryInto, vec::Vec};
+
+#[cfg(test)]
+mod tests;
+
+#[cfg(test)]
+mod mock;
+
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking;
+
+pub use pallet::*;
+pub mod weights;
+pub use weights::*;
+
+#[frame_support::pallet]
+pub mod pallet {
+	use super::*;
+	use common_primitives::schema::{Schema, SchemaId};
+	use frame_support::pallet_prelude::*;
+	use frame_system::pallet_prelude::*;
+
+	#[pallet::config]
+	pub trait Config: frame_system::Config {
+		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
+		type WeightInfo: WeightInfo;
+
+		// TODO: may need to be moved into MSA and import traits into schemas pallet.
```

Comment:
I'm not sure what is the use case for this. But in case of consuming MSA pallet we have a `AccountProvider` trait that can be used to get msa accounts from keys and it can easily extended for new functionalities.

I think we will need a new issue for any of existing todos to be able to track

Reply:
This was meant to be removed anyway.